### ISSUE TITLE
oui-datagrid: typeOptions not understood with dynamic columns

### ISSUE THREAD

**Issue by marie-j** [ovh-ux/ovh-ui-angular#216]
It seems that `typeOptions` is not understood when creating dynamic columns 
Example : 
```
columns="[{
  title: 'title,
  property: 'property',
  filterable: true,
  type: 'options',
  typeOptions: {
    values: {
      value1: 'value1',
      value2: 'value2'
    }
  }
 }];
```
The `Value` select is empty when displayed

**Comment by whispyy** [comment]
Hello, 
Thanks for the report.
I've made a fix for this :)
I'm about to provide a more precise test about this use case!

**Comment by AxelPeter** [comment]
This issue will be fixed in the release v2.16.2
